### ISSUE TITLE
New feature: drummer.this.how

### ISSUE THREAD

**Issue by scripting** [scripting/drummerSupport#105]
See this change note.

**Comment by mistersugar** [comment]
Thank you for this feature. I have tried it successfully for my placesLived.opml public outline. The page is at http://drummer.this.how/mistersugar/placesLived.opml

Does drummer.this.how reflect instant outlining?

I have the same outline viewable through the previous testing feature Concord Reader at http://scripting.com/code/testing/concordreader/?url=http://drummer.scripting.com/mistersugar/placesLived.opml 

That does reflect instant outlining.

So, two good options.

**Comment by scripting** [comment]
@mistersugar -- no, it doesn't instantly update, it certainly could. 

sometimes when you go for a picnic you forget the mustard, it's one of those things. ;-)

actually it's more like when this software was written instant outlining wasn't as solid as it is now. 

it'll go on the list for sure.

**Comment by scripting** [comment]
@mistersugar -- it's in. https://www.youtube.com/watch?v=hhLrVZrZi24

Thanks for the suggestion. :boom:

**Comment by mistersugar** [comment]
Glad the suggestion was a good one and it was easy to implement. I can confirm I am seeing http://drummer.this.how/mistersugar/placesLived.opml reflect instant outlining.

Out of curiosity, I run Page Park. Can I serve my own standalone pages?

**Comment by scripting** [comment]
@mistersugar -- yes -- it should work as-is. any opml file served by pagepark is served that way.

**Comment by fmcpherson** [comment]
You asked for comments.

I've been doing this with Little Outliner for a while. I imported an outline I created with LO in to Drummer and it rendered out of the box: http://drummer.this.how/frankm/2020BooksRead.opml. Another good use of it is here: http://my.this.how/frankm/techThatIUse.opml  What if clicking the page title toggled expand all/collapse all? 

Challenge is, one might not always want the parent nodes of an outline to render as a collapsable head. I might want the OPML to render those nodes as standard HTML H2 heads. Don't know how hard it would be, but what I think would be cool is if PagePark could support both, say by the user specifying which in a OPML head?  Just an idea.
